python/tank/path_cache.py

```diff
@@ -1017,7 +1058,8 @@ def get_folder_tree_from_sg_id(self, shotgun_id):
         root_name = res[0][0]
```

Comment:
ln 1057 (expand this block up) comment is now incorrect since you're using `root_path` instead of `root_name`